src/ProxyInterfaceSourceGenerator/Utils/MemberHelper.cs

```diff
@@ -79,22 +79,24 @@ internal static class MemberHelper
     // TODO : do we need also to check for "SanitizedName()" here?
     private static IReadOnlyList<T> GetPublicMembers<T>(
         ClassSymbol classSymbol,
-        bool proxyBaseClasses,
+        ProxyData proxyData,
         params Func<T, bool>[] filters
     ) where T : ISymbol
     {
         var membersQuery = classSymbol.Symbol.GetMembers().OfType<T>()
             .Where(m => m.DeclaredAccessibility == Accessibility.Public);
 
-        foreach (var filter in filters)
+        var f = filters.ToList();
+        f.Add(x => !proxyData.MembersToIgnore.Contains(x.Name));

```

Comment:
Instead of just use string Contains, I think it's a better design to let the user define a regular expression.

So rename `MembersToIgnore` to `MemberRegularExpressionsToIgnore` ? 

This makes the usage more flexible.

Reply:
Not sure Regex will help as members usually don't have similar names.  Also, I feel that Regexes make things less clear.   Happy to look at alternatives though

Reply:
Question: will this only work for properties or also for methods?

Reply:
works for all members...the filter is the final thing that all members use